setup.cfg

```diff
@@ -68,7 +69,6 @@ dev =
     urllib3==1.26.17
     wheel
     yapf
-    python-semantic-release==7.33.2
```

Comment:
Ok, good catch!! Back to the drawing board here. The hope was to eliminate a set of warnings simply by getting rid of `python-semantic-release`. What wasn't considered was the fact that trestle `make release` won't work if this is removed, which wasn't noticed because `make release` is not part of the test harness! How dumb? Please don't answer :-)

And by the way, these warnings already occur in the current develop branch, so there is no regression if the warnings reappear. The warnings are for removal of support for this level of semantic-release in python 3.12. However, we upgraded support to 3.9, 3.10, 3.11 in this PR, but not yet 3.12, so that is not a show stopper.

Reply:
Changing to the latest version of `python-semantic-release` requires a change in the CICD pipeline as it no longer contains the functionality to upload to pypi. Also moving to the latest release requires moving to `pydantic>=2` which is a breaking change with the trestle code.

Reply:
In this PR `pydantic[email]`, meaning the latest version of pydantic is pulled. But, in the trestle code where pydantic is used for example `import pydantic.v1.networks`. The thought here was that we use the latest pydnatic publications, but for now use the v1 interface. The thought being that support and CVE fixes for older publications may not be forthcoming.

Reply:
It is not clear how difficult it would be to upgrade the trestle CICD pipeline to employ a newer version of semantic-release. I can investigate, but if it becomes too much of a chore (say, not finished by end of this week) perhaps we should back off and live with the warnings in order to get support for OSCAL 1.1.2 out the door. Also, anyone with more expertise willing to help on this topic (e.g. making trestle CICD pipeline changes needed) is welcomed!

Reply:
@degenaro let me know if you need a hand. You will probably want to do this on a fork